<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCB Vídeo Auto Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Arquivo de origem</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Selecione em arquivo</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEditVideoIn"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonOpenFile">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Arquivos de origem</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QListView" name="listViewVideosIn"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDeleteVideoIn">
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Pré-visualização do vídeo</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QWidget" name="widgetPreview" native="true">
               <property name="styleSheet">
                <string notr="true">border: 1px solid white;
border-radius: 10px</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>200</height>
                </size>
               </property>
               <property name="title">
                <string>Edição automática</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_11">
                    <property name="sizeConstraint">
                     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
                    </property>
                    <item>
                     <widget class="QCheckBox" name="checkBoxEnableEdit">
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Habilitar</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Mín. áudio (%):</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBoxMinAudio">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Mín. Movimento (%):</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBoxMinMotion">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_7">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="title">
                     <string>Momentos exclusos</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                         <item>
                          <widget class="QLabel" name="label_4">
                           <property name="text">
                            <string>Início,Fim:</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLineEdit" name="lineEditStartentSegment">
                           <property name="placeholderText">
                            <string>1sec,10sec 20sec,30sec</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QToolButton" name="toolButtonStartSegment">
                           <property name="text">
                            <string>[</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QToolButton" name="toolButtonEndSegment">
                           <property name="text">
                            <string>]</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QToolButton" name="toolButtonaddSegment">
                           <property name="text">
                            <string>+</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QListView" name="listViewSegments"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Vídeo</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QCheckBox" name="checkBoxEnableVideoConv">
               <property name="text">
                <string>Converte</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Codec:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditVideoCodec">
                 <property name="text">
                  <string>libx264</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>CRF:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxVideoCrf">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                 <property name="value">
                  <number>22</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>FPS:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxVideoFps">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="value">
                  <number>15</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Áudio</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBoxEnableAudioConv">
               <property name="text">
                <string>Converte</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Codec:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditAudioCodec">
                 <property name="text">
                  <string>aac</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Bitrate (k):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxAudioBitrate">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>320</number>
                 </property>
                 <property name="value">
                  <number>128</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Comandos finais</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditCmds">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonProcess">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 10px</string>
             </property>
             <property name="text">
              <string>Processar Vídeo!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditVideoIn</tabstop>
  <tabstop>toolButtonOpenFile</tabstop>
  <tabstop>listViewVideosIn</tabstop>
  <tabstop>pushButtonDeleteVideoIn</tabstop>
  <tabstop>checkBoxEnableEdit</tabstop>
  <tabstop>spinBoxMinAudio</tabstop>
  <tabstop>spinBoxMinMotion</tabstop>
  <tabstop>lineEditStartentSegment</tabstop>
  <tabstop>toolButtonStartSegment</tabstop>
  <tabstop>toolButtonEndSegment</tabstop>
  <tabstop>toolButtonaddSegment</tabstop>
  <tabstop>listViewSegments</tabstop>
  <tabstop>checkBoxEnableVideoConv</tabstop>
  <tabstop>lineEditVideoCodec</tabstop>
  <tabstop>spinBoxVideoCrf</tabstop>
  <tabstop>spinBoxVideoFps</tabstop>
  <tabstop>checkBoxEnableAudioConv</tabstop>
  <tabstop>lineEditAudioCodec</tabstop>
  <tabstop>spinBoxAudioBitrate</tabstop>
  <tabstop>plainTextEditCmds</tabstop>
  <tabstop>pushButtonProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
